images/kcp-upgrade/upgrade.sh

```diff
@@ -23,7 +23,7 @@ SCRIPT_DIR="$(
   pwd
 )"
 
-CONFIG="$(dirname "$(dirname "$SCRIPT_DIR")")/config/config.yaml"
+CONFIG="$(dirname "$(dirname "$SCRIPT_DIR")")/ckcp/config.yaml"
```

Comment:
It's under `/developer` now.
```suggestion
CONFIG="$(dirname "$(dirname "$SCRIPT_DIR")")/developer/ckcp/config.yaml"
```

Reply:
Done